.github/workflows/publish.yml

```diff
@@ -0,0 +1,35 @@
+name: Publish artifacts
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: "Target tag of publish"
+        required: true
+        default: 'default'
+        type: string
+
+jobs:
+  publish:
+    name: Publish artifacts with Gradle
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.inputs.tag }}
```

Comment:
when the input is wrong

<img width="1221" alt="스크린샷 2024-07-30 15 36 03" src="https://github.com/user-attachments/assets/d06c6d3a-eba1-4d7e-894a-a09ffd5903a4">